### PR TITLE
cmd/swarm: add cli UI for tag progress on pushsync

### PR DESCRIPTION
**Pull request by acud**
This PR adds some progress bars that are shown on the console when uploading files with the `--track` option.
We should also consider that right now there is no distinction between pushsync uploads and pullsync uploads, so if a user does not have pushsync enabled and uploads with this flag, the CLI will hang indefinitely because the associated tag is not updated.

We should consider integrating tags into pull syncing as well.

### CODE REVIEW

**Comment by zelig** [comment]
lint fails on travis https://travis-ci.org/ethersphere/swarm/jobs/589438833#L491

**Review by nonsense — APPROVED** [review]
~I think this can be simplified further as discussed, but good enough for a first version in my opinion.~

I see you already did that. Thanks!

File: cmd/swarm/upload.go [hunk 1]
```diff
@@ -55,12 +73,17 @@ func upload(ctx *cli.Context) {
 		defaultPath     = ctx.GlobalString(SwarmUploadDefaultPath.Name)
 		fromStdin       = ctx.GlobalBool(SwarmUpFromStdinFlag.Name)
 		mimeType        = ctx.GlobalString(SwarmUploadMimeType.Name)
+		debug           = ctx.GlobalBool("debug")
```
Comment (nonsense) [inline cmd/swarm/upload.go]: This is a bit hidden and doesn't appear in the `--help` section from `swarm up`.

**Review by zelig — APPROVED** [review]

File: cmd/swarm/upload.go [hunk 2]
```diff
@@ -28,23 +28,41 @@ import (
...
-	Description:        "uploads a file or directory to swarm using the HTTP API and prints the root hash",
-}
+var (
+	upCommand = cli.Command{
+		Action:             upload,
+		CustomHelpTemplate: helpTemplate,
+		Name:               "up",
+		Usage:              "uploads a file or directory to swarm using the HTTP API",
+		ArgsUsage:          "<file>",
+		Flags:              []cli.Flag{SwarmEncryptedFlag, SwarmPinFlag, SwarmNoTrackUploadFlag},
```
Comment (nonsense) [inline cmd/swarm/upload.go]: @acud I think the `debug` flag belongs here with a help section, no?

**Pull request merged**